### PR TITLE
pkg/actor(ticdc): stable tests by extending timeout

### LINKED ISSUE #3963: TestBroadcast is unstable
### Which jobs are flaking?

https://github.com/pingcap/tiflow/pull/3935 

### Which test(s) are flaking?

TestBroadcast

### Jenkins logs or GitHub Actions link

```log
[2021-12-17T10:31:44.215Z] 

[2021-12-17T10:31:44.215Z] === FAIL: pkg/actor TestBroadcast (5.94s)

[2021-12-17T10:31:44.215Z]     system_test.go:140: Timed out

[2021-12-17T10:31:44.215Z] 
```

https://ci.pingcap.net/blue/organizations/jenkins/atom-ut/detail/atom-ut/5678/pipeline

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed and how it works?

Close #3963 

### Check List 

Tests 

 - Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: pkg/actor/system_test.go [hunk 1]
```diff
@@ -220,7 +220,7 @@ func TestActorSendReceive(t *testing.T) {
 		t.Fatal("Timed out")
 	}
 
-	wait(t, time.Second, func() {
```
Comment (amyangfei) [inline pkg/actor/system_test.go:223]: Please add some comments, otherwise the magic number makes it hard to maintain in the feature.

**Review by amyangfei — APPROVED** [review]

File: pkg/actor/system_test.go [hunk 2]
```diff
@@ -128,15 +128,16 @@ func TestRouterSendAndSendB(t *testing.T) {
...
+func wait(t *testing.T, f func()) {
 	wait := make(chan int)
 	go func() {
 		f()
 		wait <- 0
 	}()
 	select {
 	case <-wait:
-	case <-time.After(timeout):
+	case <-time.After(5 * time.Second):
```
Comment (sdojjy) [inline pkg/actor/system_test.go:139]: why use the hardcoded time here?  is it better to extend the timeout on the function call?

Comment (overvenus) [reply pkg/actor/system_test.go:139]: The `wait` is specialized in actor testing, 5 seconds are longer enough for most cases, we can pass timeout when it's needed in future.

**Review by sdojjy — APPROVED** [review]
LGTM

**Pull request merged**